### ISSUE TITLE
API throws a 404 on /

### ISSUE THREAD

**Issue by Hobbsee** [flapjack/flapjack#781]
At the moment, version 1 and 2 of flapjack throw a 404 sinatra page for their front page.  Is this intentional, or should we show a 200 OK page to people to indicate that the API is up?

```
flapjack ➤ curl http://localhost:3081/
<!DOCTYPE html>
<html>
<head>
  <style type="text/css">
  body { text-align:center;font-family:helvetica,arial;font-size:22px;
    color:#888;margin:20px}
  #c {margin:0 auto;width:500px;text-align:left}
  </style>
</head>
<body>
  <h2>Sinatra doesn&rsquo;t know this ditty.</h2>
  <img src='http://localhost:3081/__sinatra__/404.png'>
  <div id="c">
    Try this:
    <pre># in json_params_parser.rb
class Flapjack::Gateways::JSONAPI
  get '/' do
    "Hello World"
  end
end
</pre>
  </div>
</body>
</html>
```

**Comment by ali-graham** [comment]
It would be nice not to show HTML error pages on a putatively JSON endpoint (and we may not have in the past, it tends to get mucked around with when debugging though). I would say there's no resource at that endpoint, so 404 is a valid response -- we should nominate a specific named endpoint for checking the API's presence if one's needed (e.g. "/identify", and get uptime etc. like the jabber bot command).

**Comment by Hobbsee** [comment]
It seems like /identify would be a great thing for a monitoring tool to use, to check that the API, is in fact up.

**Comment by ali-graham** [comment]
I'd like to try to have "interface consistency", i.e. similarly named endpoints/paths between the different interfaces, be that CLI/API/Web/etc. Might be tricky :smile:

**Comment by jessereynolds** [comment]
Perhaps 'GET /' should return a json document containing a link to the API's documentation?

**Comment by jessereynolds** [comment]
+1 to the 'GET /identify' command

**Comment by jessereynolds** [comment]
Is there a convention with Swagger enabled APIs to provide a swagger UI at / ?

**Comment by Hobbsee** [comment]
Yes, it is normally an interface with the endpoints, allowing you to query the api from the browser.

**Comment by ali-graham** [comment]
Can you point me to some sites that implement this, whether with swagger-ui or something different? It seems odd to mix HTML and JSON from the same port.

I have the swagger spec JSON file being served from /doc at present, and could move that to / if desired.
